cli/cmd/install_test.go

```diff
@@ -0,0 +1,118 @@
+package cmd
+
+import (
+	"os"
+	"path"
+	"runtime"
+	"sort"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInstall(t *testing.T) {
+	configs := []struct {
+		name      string
+		config    string
+		wantFiles []string
+	}{
+		{
+			name:   "sync_success_sourcev1_destv0",
+			config: "sync-success-sourcev1-destv0.yml",
+			wantFiles: []string{
+				"cloudquery.log",
+				"plugins/destination/cloudquery/test/v2.1.2/plugin",
+				"plugins/destination/cloudquery/test/v2.1.2/plugin.zip",
+				"plugins/source/cloudquery/test/v2.0.3/plugin",
+				"plugins/source/cloudquery/test/v2.0.3/plugin.zip",
+			},
+		},
+		{
+			name:   "multiple_sources",
+			config: "multiple-sources.yml",
+			wantFiles: []string{
+				"cloudquery.log",
+				"plugins/destination/cloudquery/test/v2.2.7/plugin",
+				"plugins/destination/cloudquery/test/v2.2.7/plugin.zip",
+				"plugins/source/cloudquery/test/v3.1.7/plugin",
+				"plugins/source/cloudquery/test/v3.1.7/plugin.zip",
+			},
+		},
+		{
+			name:   "multiple_destinations",
+			config: "multiple-destinations.yml",
+			wantFiles: []string{
+				"cloudquery.log",
+				"plugins/destination/cloudquery/test/v2.1.0/plugin",
+				"plugins/destination/cloudquery/test/v2.1.0/plugin.zip",
+				"plugins/destination/cloudquery/test/v2.2.7/plugin",
+				"plugins/destination/cloudquery/test/v2.2.7/plugin.zip",
```

Comment:
I believe these versions get updated automatically by bots (https://github.com/cloudquery/cloudquery/blob/main/cli/cmd/testdata/multiple-destinations.yml#L19) so we will need to read them from the config or keep them in sync some other way for those automated PRs to succeed

Reply:
This is the regex we use https://github.com/cloudquery/.github/blob/c43e6ce1c7e117aa0da14aff24ec15df002fe0a6/.github/renovate-default.json5#L87 so if you create a new spec without the comment it won't automatically update the versions

Reply:
For example https://github.com/cloudquery/cloudquery/blob/54cecdc91bb1af0501ac2ff4f65b4a4a448305bc/cli/cmd/testdata/multiple-destinations.yml#L6 pins the source version